pyspark/test/bigdl/test_utils.py

```diff
@@ -173,13 +173,27 @@ def __generate_keras_model(self, functional_api, input_data, output_layer):
             keras_model = self.__generate_sequence(input_data, output_layer)
         return keras_model
 
+    def __generate_random_weights(self, weights):
+        # weights: a list of ndarray; weights from keras
+        # Randomly generate a new list with the same shape to serve as the new weights for testing
+        new_weights = []
+        for weight in weights:
+            new_weights.append(np.random.random(list(weight.shape)))
+        return new_weights
+
     def modelTest(self,
                   input_data,
                   keras_model,
+                  random_weights=True,
                   dump_weights=False,
                   is_training=False,
-                  rtol=1e-7,
-                  atol=1e-7):
+                  rtol=1e-6,
+                  atol=1e-6):
+        if random_weights:
```

Comment:
why we need this option?

Reply:
To better test the weight converter as we discussed before. Since for default keras weights, some entries are zero and if some weights are not correct, test still pass.